geomesa-index-api/src/main/scala/org/locationtech/geomesa/index/index/s3/package.scala

```diff
@@ -29,10 +25,12 @@ package object s3 {
     }
   }
 
-  case class S3IndexValues(sfc: S3SFC,
-                           geometries: FilterValues[Geometry],
-                           spatialBounds: Seq[(Double, Double, Double, Double)],
-                           intervals: FilterValues[Bounds[ZonedDateTime]],
-                           temporalBounds: Map[Short, Seq[(Int, Int)]],
-                           temporalUnbounded: Seq[(Short, Short)])
+  case class S3IndexValues(
+      sfc: S2SFC,
+      maxTime: Int,
```

Comment:
What does adding the maxTime allow for?

Reply:
previously the S3SFC class had a max time, but it wasn't used in the SFC calculations at all - it was just using the s2 values, so I moved it out of the SFC to here and removed the S3SFC as redundant with the S2SFC.